eng/pipelines/test-integration-job.yml

```diff
@@ -47,9 +47,9 @@ steps:
     condition: not(succeeded())
 
   - task: PublishBuildArtifacts@1
-    displayName: Publish Screenshots
+    displayName: Publish Screenshots and Test Attachments
     inputs:
-      PathtoPublish: '$(Build.SourcesDirectory)\artifacts\bin\Microsoft.VisualStudio.LanguageServices.IntegrationTests\${{ parameters.configuration }}\net472\xUnitResults'
+      PathtoPublish: '$(Build.SourcesDirectory)\artifacts\bin\Microsoft.VisualStudio.LanguageServices.IntegrationTests\${{ parameters.configuration }}\net472\TestResults'
       ArtifactName: '$(System.JobAttempt)-Screenshots ${{ parameters.configuration }} OOP64_${{ parameters.oop64bit }} OOPCoreClr_${{ parameters.oopCoreClr }} LspEditor_${{ parameters.lspEditor }} $(Build.BuildNumber)'
```

Comment:
We may want to rename the artifact in this case, but just want to see if this works first.